runtime/interpreter/encoding_test.go

```diff
@@ -425,12 +470,15 @@ func TestEncodeDecodeComposite(t *testing.T) {
 					0x1,
 					// key 1
 					0x1,
-					// UTF-8 string, length 18
-					0x72,
-					// A.0x1.SimpleStruct
+					// UTF-8 string, length 31
+					0x78, 0x1F,
+					// A.0000000000000001.SimpleStruct
 					0x41,
-					0x2E, 0x30, 0x78, 0x31,
-					0x2E, 0x53, 0x69, 0x6d, 0x70, 0x6c, 0x65, 0x53, 0x74, 0x72, 0x75, 0x63, 0x74,
+					0x2E,
+					0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30,
+					0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x30, 0x31,
+					0x2E,
+					0x53, 0x69, 0x6d, 0x70, 0x6c, 0x65, 0x53, 0x74, 0x72, 0x75, 0x63, 0x74,
```

Comment:
The type identifier here was incorrect, so I fixed it (the type ID is `A.0000000000000001.SimpleStruct`, not `A.0x1.SimpleStruct`)